.editorconfig

```diff
@@ -1,12 +0,0 @@
-# top-most EditorConfig file
```

Comment:
Removing .editcofig?
As far as I remember have already did it couple days ago ;)